src/sentry/static/sentry/app/components/sidebar/onboardingStatus.jsx

```diff
@@ -16,6 +17,23 @@ class OnboardingStatus extends React.Component {
     collapsed: PropTypes.bool,
   };
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
Damnit I gave semi-poor advice, we'll want to use `componentDidUpdate` (which gets passed previous props, instead of next props) to future proof a bit because they're planning on deprecating `cWRP`.